0.data-download/scripts/nbconverted/data_downloader.py

```diff
@@ -33,7 +33,7 @@ def download_dependency_data(figshare_id, figshare_url, output_file):
 output_dir = pathlib.Path("data")
 figshare_url = "https://ndownloader.figshare.com/files/"
 
-download_dict = {"34990033": "CRISPRGeneDependency.csv", "35020903": "Model.csv"}
+download_dict = {"40448549": "CRISPRGeneDependency.csv", "40448834": "Model.csv"}
```

Comment:
Can you add a note somewhere in the README describing the DepMap release (please make the edits in the corresponding `.ipynb` files, FYI)

Reply:
Yep, will update README to 23Q2